app_dart/bin/server.dart

```diff
@@ -49,6 +49,7 @@ Future<void> main() async {
     final Scheduler scheduler = Scheduler(
       cache: cache,
       config: config,
+      luciBuildService: luciBuildService,
```

Comment:
Would the use of luciBuildService be encapsulated inside the scheduler?

Reply:
Yes, the goal is these are internal services that request handlers shouldn't know about. This helps minimize the logic in the handlers